test/index.js

```diff
@@ -44,7 +44,7 @@ function run(files, force) {
     const expectationPath = path.join(dirname, basename + '.pdf')
     const resultPath      = path.join(dirname, basename + '.result.pdf')
 
-    const script = require(path.join('../', scriptPath))
+    const script = require(scriptPath)
```

Comment:
Did you made this change to be able to run "npm test pdfs/text/append.js" instead of "npm test test/pdfs/text/append.js"? I am wondering whether we should stick with the latter, because of being able to use file autocomplete in the terminal?

Reply:
Good point with autocomplete... The `scriptPath` param is already absolute, so I had to remove the relative path.join. However it seems to me that the best solution to support autocomplete is to change [this](https://github.com/rkusa/pdfjs/pull/78/files#diff-910eb6f57886ca16c136101fb1699231R12) to 
```js
run(args.map((a) => path.join(__dirname, '../', a)), true)
```
I've tested it on both os and it works nicely. I would push the change if you agree.

Reply:
Sorry, I must have missed the notification here! :-(
The suggested change sounds good to me!